pandas/tests/test_groupby.py

```diff
@@ -107,6 +107,15 @@ def setUp(self):
                                       'E': np.random.randn(11),
                                       'F': np.random.randn(11)})
 
+        index = MultiIndex(levels=[[1, 2], [1, 2]],
+                           labels=[[0, 0, 0, 0, 1, 1], [1, 1, 0, 0, 0, 0]],
+                           names=['a', 'b'])
+        self.mseries = Series([0, 1, 2, 3, 4, 5], index=index)
+        index = MultiIndex(levels=[[1, 2], [1, 2]],
+                           labels=[[0, 0, 1], [1, 0, 0]],
```

Comment:
actually
move these into the actual test function
they only need to be created once
if they are in the setup they will be for every test - which is not necessary